src/components/Todo/todo.jsx

```diff
@@ -25,17 +31,26 @@ const Todo = () => {
     dispatch(handleCreateButton(isCreateButtonClicked));
   };
 
+  const handleCancelClick = () => {
```

Comment:
see https://github.com/azo-wtag/todo-app-react/pull/4/files#r1172078490

```suggestion
  function handleCancelClick() {
```